framework/include/utils/Conversion.h

```diff
@@ -67,11 +67,17 @@ namespace Moose
     return os.str();
   }
 
-  /**
-   * Convert solve type into human readable string
-   */
+  /// Convert solve type into human readable string
   template<>
   std::string stringify(const SolveType & t);
+
+  /// Convert solve type into human readable string
```

Comment:
Fix this doco string while you're at it, looks like it was copied from the function above...